src/main/java/seedu/manager/command/ViewCommand.java

```diff
@@ -27,23 +28,20 @@ public ViewCommand(String eventName) {
      * @return The command output with a list message
      */
     public CommandOutput execute() {
-        Event event = null;
-        for (int i = 0; i < eventList.getListSize(); i++) {
-            if (eventList.getEvent(i).getEventName().equals(eventName)) {
-                event = eventList.getEvent(i);
-            }
-        }
+        try {
+            Event eventToView = eventList.getEventByName(this.eventName);
 
-        //to handle errors in the future
+            StringBuilder outputMessage = new StringBuilder(
+                    String.format(VIEW_MESSAGE, eventToView.getParticipantCount(), eventName) + "\n");
+            int count = 1;
+            for (String participant : eventToView.getParticipantList()) {
+                outputMessage.append(String.format("%d. %s\n", count, participant));
+                count++;
```

Comment:
Perhaps you could consider directly using count++ in the String.format(), there's no need to do increment outside it.